web/src/pm_list.js

```diff
@@ -202,6 +226,17 @@ function zoom_out() {
     $(".private_messages_container").removeClass("zoom-in").addClass("zoom-out");
     $("#streams_list").show();
     $(".left-sidebar .right-sidebar-items").show();
+
+    // Remove filter section for direct messages.
+    $(".filter-pms").remove();
```

Comment:
Can we write this as `$(".pm_filter_row .filter-pms").remove();` for closer symmetry with the above?

Even better would be something like `$(".pm_filter_row").empty();`

Reply:
Updated!